app/views/catalog/_field_list.html.erb

```diff
@@ -1,10 +1,22 @@
-<dl>
-  <% fields.each do |method| 
-    asset.send(method).tap do |value| 
-      if !value.empty? %>
-        <dt><%= method.gsub('_',' ').gsub(/clip /i, '') %></dt>
-        <dd><%= value %></dd><% 
-      end
-    end
-  end %>
-</dl>
\ No newline at end of file
+<%
+# default to 1 column, and only allow up to 4
+columns ||= 1
+columns = [columns.to_i, 4].min
```

Comment:
If I did specify 6, I think it'd be better just to have it fail quickly, than to have it drop to 4, and I'm left wondering what's up.